runtime/web_worker.rs

```diff
@@ -36,38 +37,83 @@ use std::task::Context;
 use std::task::Poll;
 use tokio::sync::Mutex as AsyncMutex;
 
+pub type WorkerId = u32;
```

Comment:
Consider turning this into a newtype (`pub struct WorkerId(u32);`) if you're worried about people mixing up ids with other integral values.

Reply:
Fixed it